system-addon/lib/SnippetsFeed.jsm

```diff
@@ -27,12 +33,22 @@ this.SnippetsFeed = class SnippetsFeed {
   constructor() {
     this._refresh = this._refresh.bind(this);
   }
+
   get snippetsURL() {
     const updateURL = Services
       .prefs.getStringPref(SNIPPETS_URL_PREF)
       .replace("%STARTPAGE_VERSION%", STARTPAGE_VERSION);
     return Services.urlFormatter.formatURL(updateURL);
   }
+
+  isDefaultBrowser() {
+    try {
+      return ShellService.isDefaultBrowser();
+    } catch (e) {}
+    // istanbul ignore next
+    return null;
```

Comment:
There needs to be a distinction between `true`, `false`, and we couldn't actually determine the value (this is also copying what the existing API is)